ftplugin/k8s_jobs.lua

```diff
@@ -0,0 +1,71 @@
+local api = vim.api
+local commands = require("kubectl.actions.commands")
+local definition = require("kubectl.views.jobs.definition")
+local job_view = require("kubectl.views.jobs")
+local loop = require("kubectl.utils.loop")
+local root_view = require("kubectl.views.root")
+local view = require("kubectl.views")
+
+--- Set key mappings for the buffer
+local function set_keymaps(bufnr)
+  api.nvim_buf_set_keymap(bufnr, "n", "g?", "", {
+    noremap = true,
+    silent = true,
+    desc = "Help",
+    callback = function()
+      view.Hints(definition.hints)
+    end,
+  })
+
+  api.nvim_buf_set_keymap(bufnr, "n", "<CR>", "", {
+    noremap = true,
+    silent = true,
+    desc = "Go to pods",
+    callback = function()
+      local name, ns = job_view.getCurrentSelection()
+      view.set_and_open_pod_selector("jobs", name, ns)
+    end,
+  })
+
+  api.nvim_buf_set_keymap(bufnr, "n", "<bs>", "", {
+    noremap = true,
+    silent = true,
+    desc = "Go up",
+    callback = function()
+      root_view.View()
+    end,
+  })
+
+  api.nvim_buf_set_keymap(bufnr, "n", "grr", "", {
```

Comment:
Should we use a different keybind?
Grr made sense for rollout restart but not so much here

Reply:
yeah i just had no clue what to use. suggestions?

Reply:
k9s seems to call it trigger, that's not that bad, also gc (create) could be an option?